lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -969,8 +970,12 @@ data ByronWalletPutPassphraseData = ByronWalletPutPassphraseData
     , newPassphrase :: !(ApiT (Passphrase "user"))
     } deriving (Eq, Generic, Show)
 
+data ApiSealedTxEncoding = HexEncoded | Base64Encoded
+      deriving (Eq, Generic, Show)
+      deriving anyclass NFData
+
 data ApiConstructTransaction (n :: NetworkDiscriminant) = ApiConstructTransaction
-    { transaction :: !(ApiT SealedTx)
+    { transaction :: !(ApiT SealedTx, ApiSealedTxEncoding)
```

Comment:
```suggestion
    { transaction :: ApiSerialisedTransaction
```

I agree with [Yura's comment on `ApiSerialisedTransaction`](https://github.com/input-output-hk/cardano-wallet/pull/3449#discussion_r950125122) here.

The many pattern matches

```hs
         let (txCbor,_) = getFromResponse #transaction rTx
```

can be avoided in favor of extracting a subfield,

```
         let txCbor = getFromResponse (#transaction . #sealedTx) rTx
```

or something like that. One could even use `Quantity`, but I don't want to drag that type into this.